.drone.yml

```diff
@@ -61,19 +61,14 @@ steps:
   pull: if-not-exists
   image: plugins/s3
   settings:
-    access_key: ${AWS_ACCESS_KEY_ID}
-    bucket: ${S3_BUCKET_NAME}
+    bucket: cs-integration-testing-s3
     region: eu-west-2
-    secret_key: ${AWS_SECRET_ACCESS_KEY}
-    source: target/site/serenity/**/*
-    target: /reports/$${DRONE_BUILD_NUMBER}/
-  environment:
-    AWS_ACCESS_KEY_ID:
+    access_key:
       from_secret: aws_access_key_id
-    AWS_SECRET_ACCESS_KEY:
+    secret_key:
       from_secret: aws_secret_access_key
-    S3_BUCKET_NAME:
-      from_secret: s3_bucket_name
+    source: /drone/src/target/site/serenity/**/*
+    target: /reports/$${DRONE_BUILD_NUMBER}/
```

Comment:
Will this report be places at the correct folder as the source has changed? and will that impact?

Reply:
According to the logs from "execute-tests" the report is written to /drone/src/target/site/serenity

Reply:
Not to worry then - looks good to me - think Douglas mentioned setting the workspace to `/drone/src`. But that can always be done later for me 👍🏼

Reply:
I agree, we can neaten it up later. I just need to get it working first.